### ISSUE TITLE
Input images don't display in dataloader and test_preds images

### ISSUE THREAD

**Issue by lewfish** [azavea/raster-vision#1132]
When running the `tiny_spacenet.py` example in 0.13 with `rastervision run local rastervision.pytorch_backend.examples.tiny_spacenet`, the input images are blank in `train/dataloaders/*.png` and `train/test_preds.png`. 

![train](https://user-images.githubusercontent.com/1896461/110862984-01acc980-828e-11eb-8e25-d97b08a2214b.png)

![test_preds](https://user-images.githubusercontent.com/1896461/110862995-04a7ba00-828e-11eb-80e1-1fe83aa58cde.png)

**Comment by AdeelH** [comment]
This seems to be a consequence of removing the `StatsTransformer`. The data is not in the range 0-1 when it gets to `Learner.plot_batch()`.

One possible remedy would be to detect if the data is not in the right format for matplotlib, automatically do a default transformation to normalize it, and throw a warning telling the user to specify a custom `plot_transform` in `DataConfig.PlotOptions` for best results.

**Comment by lewfish** [comment]
> One possible remedy would be to detect if the data is not in the right format for matplotlib, automatically do a default transformation to normalize it, and throw a warning telling the user to specify a custom `plot_transform` in `DataConfig.PlotOptions` for best results.

Ok, sounds good.
